src/scripts/commands/database.ts

```diff
@@ -0,0 +1,205 @@
+// Commands related to dhis2 database direct manipulation.
+// The generated sql files can be run for example with: d2-docker run-sql <file>
+
+import _ from "lodash";
+import fs from "fs";
+import { command, subcommands, positional, flag, option, optional, string } from "cmd-ts";
+import { HierarchyLevel, OrgUnitPath } from "scripts/common";
+
+export function getCommand() {
+    const removeOrgUnitLevelCommand = command({
+        name: "remove-orgunit-level",
+        description: "Create a sql file that removes organisation units below a hierarchy level",
+        args: {
+            level: positional({
+                type: HierarchyLevel,
+                description: "Maximum hierarchy level to keep",
+            }),
+            outputFile: option({
+                type: optional(string),
+                long: "output-file",
+                description: "Name of the output file that will contain the sql commands",
+            }),
+            overwrite: flag({
+                long: "overwrite",
+                description: "Overwrite ouput file if exists",
+            }),
+        },
+        handler: async args => {
+            const outputFile = args.outputFile ?? "remove_orgunits.sql";
+            const sqlCommands = removeOrgs(`hierarchylevel > ${args.level}`);
+            safeWrite(outputFile, sqlCommands, args.overwrite);
+            process.exit(0);
+        },
+    });
+
+    const removeOrgUnitChildrenCommand = command({
+        name: "remove-orgunit-children",
+        description: "Create a sql file that removes all children of an organisation unit",
+        args: {
+            path: positional({
+                type: OrgUnitPath,
+                description: "Path of the organisation unit to prune of its children",
+            }),
+            outputFile: option({
+                type: optional(string),
+                long: "output-file",
+                description: "Name of the output file that will contain the sql commands",
+            }),
+            overwrite: flag({
+                long: "overwrite",
+                description: "Overwrite ouput file if exists",
+            }),
+        },
+        handler: async args => {
+            const outputFile = args.outputFile ?? "remove_orgunits.sql";
+            const sqlCommands = removeOrgs(`path LIKE '${args.path}/%'`);
```

Comment:
These 2 sub-commands are so similar that I wonder: does it make to create a single command with different options for each filter? example: `--from-level=LEVEL` / `--by-path=PATH`. Both filters could work together (as and AND, for example)

Reply:
Yes, makes sense. This came from not being that familiar with cmd-ts. Done as part of 52f4ab802ac41ecf2049ae992b70785aa1062106